Makefile

```diff
@@ -33,10 +39,17 @@ kustomize: $(KUSTOMIZE) ## Download kustomize locally if necessary.
 
 ## test: Run unit tests
 .PHONY : test
```

Comment:
maybe just me, but seeing:

```
make test                                                                         git:(general-enhancements|⚑3)
rm -rf /Users/jmadigan/Work/kuadrantctl/coverage
go fmt ./...
go vet ./...
mkdir -p /Users/jmadigan/Work/kuadrantctl/coverage
# Shuffle both the order in which specs within a suite run, and the order in which different suites run
# You can always rerun a given ordering later by passing the --seed flag a matching seed.
/Users/jmadigan/Work/kuadrantctl/bin/ginkgo \
		--randomize-all \
		--randomize-suites \
		--coverpkg ./pkg/...,./cmd/... \
		--output-dir /Users/jmadigan/Work/kuadrantctl/coverage \
		--coverprofile cover.out \
		./pkg/... ./cmd/...
flag provided but not defined: -randomize-all
```

Reply:
Maybe I've got a different version or something,

Seeing this in `--help` for ginko:

```
  -randomizeAllSpecs
    	If set, ginkgo will randomize all specs together.  By default, ginkgo only randomizes the top level Describe, Context and When groups.
  -randomizeSuites
    	When true, Ginkgo will randomize the order in which test suites run
```

rather than `--randomize-all` & `--randomize-suites`

```
kuadrantctl/bin/ginkgo version
Ginkgo Version 1.16.4
```

Reply:
remove `bin` dir and run it again

```
kuadrantctl on  general-enhancements via 🐹 v1.21.4 vim-go ❯ rm -rf bin/
kuadrantctl on  general-enhancements via 🐹 v1.21.4 vim-go ❯ make ginkgo
# In order to make sure the version of the ginkgo cli installed
# is the same as the version of go.mod,
# instead of calling go-install-tool,
# running go install from the current module will pick version from current go.mod file.
GOBIN=/home/eguzki/git/kuadrant/kuadrantctl/bin go install github.com/onsi/ginkgo/v2/ginkgo
kuadrantctl on  general-enhancements via 🐹 v1.21.4 vim-go took 2s ❯ bin/ginkgo version
Ginkgo Version 2.13.2
```

Reply:
```
  -randomizeAllSpecs
    	If set, ginkgo will randomize all specs together.  By default, ginkgo only randomizes the top level Describe, Context and When groups.
  -randomizeSuites
    	When true, Ginkgo will randomize the order in which test suites run
```

are deprecated in v2. I got a warning message and I replaced the option params.